examples/monitors/with-payload-template/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
```

Comment:
with-payload-template is not categorically fitting this folder. so let's remove this please